### ISSUE TITLE
Create shell scripts to run "How to run" commands in README section

### ISSUE THREAD

**Issue by grdddj** [trezor/trezor-user-env#107]
We could create bash scripts like `run_linux.sh` and `run_mac.sh`, that would contain all the necessary steps how to run the `tenv` after cloning the repository.

It would be just aggregating the instructions from README, like:
```sh
xhost +
docker-compose -f ./docker/compose.yml pull trezor-user-env-unix trezor-user-env-regtest
docker-compose -f ./docker/compose.yml up trezor-user-env-unix trezor-user-env-regtest
```

Benefit would be that users would not for example forget the `xhost+` command, and would be able to really run it with one click (enter command)

**Comment by tsusanka** [comment]
I sometimes find it more useful to copy & paste instructions from README, becase you have an idea what you are doing and it is yet another thing to maintain. But I agree this might be helpful for newcomers..

**Comment by vdovhanych** [comment]
i agree with @tsusanka it could help with newcomers bu in general i like step by step guides more that just run this simple script without seeing what it does right away.

**Comment by tsusanka** [comment]
Done 🎉 .
